app/src/main/res/values/strings.xml

```diff
@@ -47,6 +47,11 @@
     </plurals>
     <string name="water_date" translatable="false">%1$s - %2$s</string>
 
+    <plurals name="garden_action_mode_count">
```

Comment:
Is this used?

Reply:
It will be. Sorry, I didn't remove it but it's used by the ActionMode which will come next.

Reply:
SG.  Can you add a comment describing having a "one" and "other" quantity even though the resulting string are the same?